src/blocks/battery.rs

```diff
@@ -87,10 +87,14 @@ impl PowerSupplyDevice {
 impl BatteryDevice for PowerSupplyDevice {
     fn is_available(&self) -> bool {
         let path = self.device_path.join("present");
-        if !path.exists() {
-            return false;
+        if path.exists() {
```

Comment:
Can you refactor both checks into one block since the only different part is the path it's checking ("present" vs "online")?

Reply:
Something like

```rust
for file in ["present", "online"] {
    // ...
}
```

Reply:
I double checked this and apparently online is not quite the same thing:
https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-power

I guess it would be better to just check if the "present" file exists and assume the device is available if it doesn't?